azure-pipelines.yml

```diff
@@ -73,12 +73,17 @@ jobs:
     vmImage: vs2017-win2016
   steps:
   - powershell: |
-      # Notify slack channel about failed courtesy push PR creation
-      $pipelineLink = "$env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI$env:SYSTEM_TEAMPROJECT/_build/results?buildId=$env:BUILD_BUILDID&_a=summary" 
-      $messageText = "Failed to create courtesy push PR. Please review the results of failed build: $($pipelineLink) "
-      $body = '{"text": "'+$($messageText)+'"}'
-      Invoke-RestMethod -Uri $(slackUri) -Method Post -Body $body -ContentType 'application/json'
-    displayName: 'Send notification to the slack channel'
+      # Notify MS Teams channel about failed courtesy push PR creation
+      $pipelineLink = "$env:SYSTEM_TEAMFOUNDATIONCOLLECTIONURI$env:SYSTEM_TEAMPROJECT/_build/results?buildId=$env:BUILD_BUILDID&_a=summary"
+      $titleText = "Courtesy push build failed - ID $($env:BUILD_BUILDID)"
+      $messageText = "Failed to create Courtesy Bump of Tasks PR. Please review the results of failed build [ID $($env:BUILD_BUILDID)]($($pipelineLink)). Related article in [Wiki](https://mseng.visualstudio.com/AzureDevOps/_wiki/wikis/AzureDevOps.wiki/25317/Release-of-pipeline-tasks)."
+      $body = [PSCustomObject]@{
+        title = $titleText
+        text = $messageText
+        themeColor = "#FF0000"
+      } | ConvertTo-Json
```

Comment:
Could you please add some comments or the link to the related docs regarding which properties can be used in the body?

Reply:
Done. Comments are added.